CHANGELOG.md

```diff
@@ -39,6 +39,13 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
   functionality of `NonAggregate`. See [the upgrade
   notes](#2-0-0-upgrade-non-aggregate) for details.
 
+### Fixed
```

Comment:
There is already a `Fixed` section in line 94. Could you move this point there?
Additionally I'm not that happy with the formulation "This allows
  applications to use in-memory databases" as it is already possible to do this using `":memory:"` as connection string passed to `SqliteConnection::establish`. Maybe just drop that point and only say that we support full sqlite URIs now?

Reply:
Done.